README.md

```diff
@@ -29,15 +29,20 @@ Why?
 
 ## Usage
 
-To clone the repository and run this website on your local machine, [install Jekyll](https://jekyllrb.com/docs/installation/) for your OS and type the following commands:
+### Dependencies
 
-    $ sudo gem install jekyll bundler
+Install [asdf-vm](https://asdf-vm.github.io/asdf/#/core-manage-asdf-vm) for your OS and now using asdf-vm, install [ruby](https://github.com/asdf-vm/asdf-ruby).
+
+### Cloning and running
```

Comment:
Maybe rename this to `Download and Run`?

Reply:
@frextrite but we are not downloading anything. should I use `Clone and Run` ?

Reply:
Yep `Clone and Run` works.